R/PKNCA.R

```diff
@@ -143,6 +143,100 @@ PKNCA_create_data_object <- function(adnca_data) { # nolint: object_name_linter
   pknca_data_object
 }
 
+
+
+#' Create a PKNCAdata Object for NCA or Slope Analysis
+#'
+#' This function updates a previously prepared `PKNCAdata` object
+#' based on user selections for method, analyte, dose, specimen, and parameters.
+#' @details
+#' Step 1: Update units in the `PKNCAdata` object
+#' ensuring unique analytes have their unique units
+#' Step 2: Set `PKNCAoptions` for NCA calculation
+#' Step 3: Format intervals using `format_pkncadata_intervals()`
+#' Step 4: Apply filtering based on user selections
+#' Step 5: Impute start values if requested
```

Comment:
nitpick: In the documentation, this will render as a single line. To give it a nice separation (so it forms a list) add empty lines inbetween each point or use 

```suggestion
#' Step 1: Update units in the `PKNCAdata` object
#' ensuring unique analytes have their unique units
#'
#' Step 2: Set `PKNCAoptions` for NCA calculation
#'
#' Step 3: Format intervals using `format_pkncadata_intervals()`
#'
#' Step 4: Apply filtering based on user selections
#'
#' Step 5: Impute start values if requested
```

or you can plant `*` before each item to create a list, or maybe use `\enumerate{}`.